.github/workflows/main-benchmark.yml

```diff
@@ -0,0 +1,63 @@
+name: Benchmark main
```

Comment:
Can we simply call this benchmark>

Reply:
this one runs only on the `main` branch, that's why I called it `Benchmark main`